irohad/main/application.cpp

```diff
@@ -172,10 +179,13 @@ void Irohad::initBatchParser() {
 void Irohad::initValidators() {
   auto factory = std::make_unique<shared_model::proto::ProtoProposalFactory<
       shared_model::validation::DefaultProposalValidator>>();
-  stateful_validator =
-      std::make_shared<StatefulValidatorImpl>(std::move(factory), batch_parser);
+  stateful_validator = std::make_shared<StatefulValidatorImpl>(
+      std::move(factory),
+      batch_parser,
+      validators_log_manager_->getChild("Stateful")->getLogger());
   chain_validator = std::make_shared<ChainValidatorImpl>(
-      std::make_shared<consensus::yac::SupermajorityCheckerImpl>());
+      std::make_shared<consensus::yac::SupermajorityCheckerImpl>(),
+      validators_log_manager_->getChild("Chain")->getLogger());
```

Comment:
The names "Chain" and "Stateful" does not seem clear for me.
I suggest using known components names.

Moreover, those names look like adjectives.
If so, please at least add nouns to the names.

Reply:
"Simulator" and "Consensus" below are OK.

Reply:
> The names "Chain" and "Stateful" does not seem clear for me.

It should be read as *get the "Chain" child of the "Validators" log node*. And in the log output it will be seen as `.../Validators/Chain/...`. We have different validators, and repeating the noun "Validator" for every child seems wrong.